src/js/conntest.js

```diff
@@ -40,32 +40,62 @@ function hostConnectivityTest() {
 }
 
 function runConnectivityTest(iceCandidateFilter, config) {
+  var candidates = [];
   var call = new Call(config);
   call.setIceCandidateFilter(iceCandidateFilter);
+
+  // Collect all candidate for validation.
+  call.pc1.onicecandidate = function(event) {
+    if (event.candidate) {
+      candidates.push(Call.parseCandidate(event.candidate.candidate));
```

Comment:
I think we need to actually check the `iceCandidateFilter` here. Also, I think it's nice to log valid candidates out with reportSuccess just like we do here: https://github.com/webrtc/testrtc/blob/master/src/js/nettest.js#L87